app/static/admin/js/pages/analytics/graph/analytics-graph.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+define(["../analytics-util", "../constants"], function(util, c) {
+
+  function Graph(elementId, keyName, valueName, startDate, endDate, color) {
```

Comment:
Continuing about making Graph data dynamic. Just a FYI again. I would take the params of elementId, keys, values, options. Where options would define things like color, labels, etc.